projects/addon-table/components/table/tbody/tbody.template.html

```diff
@@ -22,6 +21,7 @@
         </button>
     </th>
 </tr>
+<ng-content></ng-content>
```

Comment:
This way you cannot close rows by clicking heading. I guess you need to add `*ngIf="open"`

Reply:
fixed